ts/src/flexible-event/privacy.test.ts

```diff
@@ -101,6 +103,43 @@ void test('maxInformationGain', async (t) => {
   )
 })
 
+void test('epsilonToBoundInfoGainAndDp', async (t) => {
+  const infoGainUppers = [11.5, 6.5]
+  const epsilonUpper = 14
+  const numStatesRange = 100000
+  const numTests = 500
+
+  await Promise.all(
+    [...Array(numTests).keys()].map((i) =>
+      t.test(`${i}`, () => {
+        const numStates = Math.ceil(Math.random() * numStatesRange)
+        const infoGainUpper = infoGainUppers[Math.round(Math.random())] || 11.5
```

Comment:
Is the `|| 1.5` needed here? IIUC `Math.random()` returns a number in `[0, 1)`, which after rounding will be either 0 or 1, both of which are valid indices into `infoGainUppers`, meaning that the right-hand side of the `||` should never be taken.

If it's only there to satisfy TypeScript, we should instead use an assertion to indicate that we know that the value is not `undefined`:

```js
infoGainUppers[Math.round(Math.random())]!
```
If for some reason we do need to handle the case in which it's `undefined`, let's use [`??`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing) instead of `||`.

Reply:
Yes, it was a compilation error. `!` looks good, thanks!

Reply:
Done.